server/controllers/profile.js

```diff
@@ -38,4 +38,14 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route GET /profiles  
+// @desc Get user profiles
+// @access Public
+
+exports.loadProfiles = asyncHandler(async (req, res, next) => {
+  const pet_sitters = await Profile.find({ accountType: 'pet_sitter'})
```

Comment:
Since this PR also has changes pertaining to `IN: Listings Page`, let's add a couple of fields to the Profile model - price and caption/tagLine

Once done, let's make sure we query for sitter profiles who have a valid price and location along with the appropriate account type